src/resmom/Makefile.am

```diff
@@ -113,6 +113,12 @@ endif
 pbs_mom_SOURCES += linux/cpuset_misc.c
 endif
 
+if UNDOLR_ENABLED
+pbs_mom_CPPFLAGS += @libundolr_inc@
+pbs_mom_LDADD += @libundolr_lib@
+pbs_mom_SOURCES += $(top_srcdir)/src/server/undolr.c
```

Comment:
Why add undolr.c to all the daemon sources one by one? We could have added it to libutil and then it would automatically be part of all binaries (if undolr is enabled)